.buildkite/pipeline_for_upload_action_test.yml

```diff
@@ -0,0 +1,6 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+---
+
+steps:
+  - label: ":ok_hand: If this message is shown, the pipeline upload succeeded"
```

Comment:
Could benefit a custom (and more concise) `github_commit_status`.

Could be done in a separate PR though as for it to be useful we'd also want to disable the default status created by Buildkite on each step (in the `release-toolkit.tf` of `buildkite-ci`) and switch to making all steps have explicit `notify`